lua/metals.lua

```diff
@@ -307,6 +307,8 @@ M.organize_imports = function()
   for _, result in pairs(response.result or {}) do
     if result.edit then
       lsp.util.apply_workspace_edit(result.edit, "utf-16")
+    elseif result.disabled then
+      vim.notify(result.disabled.reason, vim.log.levels.WARN)
```

Comment:
```suggestion
      log.warn_and_show(result.disabled.reason)
```

I have a wrapper around `vim.notify` that I use to get things logged out, could you switch this to use that?